src/js-api/js-api.ts

```diff
@@ -1,2 +1,3 @@
-export { deploy } from "./deploy";
-export { downloadCiConfig } from "./ci-config";
+export { deploy } from "./handlers/deploy";
```

Comment:
nitpick: the term `handlers` is pretty vague. Is this folder for all the commands? Maybe rename it to `commands`?

Given that there are only 5 files in the `js-api` folder, another option could be just not to have a subfolder for now

Reply:
I change it to `commands`